bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {
+               GoRequirement(package='github.com/BurntSushi/toml/cmd/tomlv',
+                             version='0.1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Data type'}
```

Comment:
I think `Formatting` and `Syntax` is better, because the linter itself is for TOML and doesn't detect the "type"